From afcca308d444d726b6ad6dc04fed359c233d3e5f Mon Sep 17 00:00:00 2001
From: Huibin Hong <huibin.hong@rock-chips.com>
Date: Mon, 6 Jun 2016 11:05:15 +0800
Subject: [PATCH] dt/bindings: arm-pl330: Add peripherals-req-type-burst

This patch add peripherals-req-type-burst property to PL330 DMA
controller nodes to enable dma burst mode when it is present.

Change-Id: I1b6923d9a23c15be800a5a1bd3f2161dd7b8826b
Signed-off-by: Huibin Hong <huibin.hong@rock-chips.com>
---
 Documentation/devicetree/bindings/dma/arm-pl330.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/dma/arm-pl330.txt b/Documentation/devicetree/bindings/dma/arm-pl330.txt
index db7e2260f9c5..1522822b1973 100644
--- a/Documentation/devicetree/bindings/dma/arm-pl330.txt
+++ b/Documentation/devicetree/bindings/dma/arm-pl330.txt
@@ -16,6 +16,7 @@ Optional properties:
   - dma-channels: contains the total number of DMA channels supported by the DMAC
   - dma-requests: contains the total number of DMA requests supported by the DMAC
   - arm,pl330-broken-no-flushp: quirk for avoiding to execute DMAFLUSHP
+  - peripherals-req-type-burst: set peripheral dma request type as burst mode
 
 Example:
 
-- 
2.35.3

